plugins/techdocs/src/api.ts

```diff
@@ -53,10 +53,38 @@ export class TechDocsApi implements TechDocs {
     this.apiOrigin = apiOrigin;
   }
 
-  async getMetadata(metadataType: string, entityId: ParsedEntityId) {
+  /**
+   * Retrieve TechDocs metadata.
+   *
+   * When docs are built, we generate a techdocs_metadata.json and store it along with the generated
+   * static files. It includes necessary data about the docs site. This method requests techdocs-backend
+   * which retries the TechDocs metadata.
+   *
+   * @param {ParsedEntityId} entityId Object containing entity data like name, namespace, etc.
+   */
+  async getTechDocsMetadata(entityId: ParsedEntityId) {
```

Comment:
Is this the same as https://github.com/backstage/backstage/blob/master/packages/catalog-model/src/types.ts#L42 ? That one is being used pervasively throughout the codebase

Reply:
I think, yes. It's better idea to use this ^ type instead of the duplicated one defined in techdocs.

Will change in a new PR. Thanks for noticing!